src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -135,7 +137,7 @@ class Sidebar extends React.Component {
 
   hashChangeHandler = () => {
     if (window.location.hash == '#welcome') {
-      this.setState({showTodos: true});
+      this.togglePanel('todos');
```

Comment:
I don't think this was ever working, `hashChangeHandler` was never even being called either. /cc @denamwangi